src/consent.js

```diff
@@ -73,3 +74,31 @@ export function getConsentPolicyInfo(element, policyId) {
             /** @type {string} */ (policyId));
       });
 }
+
+/**
+ * Determine if an element needs to be blocked by consent based on metaTags.
+ * @param {*} element
+ * @return {boolean}
+ */
+export function shouldBlockOnConsentByMeta(element) {
+  const ampdoc = element.getAmpDoc();
+  let content =
+      Services.documentInfoForDoc(ampdoc).metaTags['amp-consent-blocking'];
+
+  if (!content) {
+    return false;
+  }
+
+  if (isArray(content)) {
```

Comment:
Looks like I can apply the uniqueness check to `<meta amp-consent-blocking>`, so should be safe removing the check here. Let me know if you also prefer that. 

In addition, any idea on the naming? Thank you